### PR TITLE
United Space Lepidoptera

### PR DESCRIPTION
**Pull request by Sibg**
## About The Pull Request
This changes UPP to USL Which in its long form is United Space Lepidoptera, this changes job names for all UPP names and also spawns them as moths. 

## Why It's Good For The Game
UPP Is a faction from Colonial marines and we should have our own factions, this being angry pirate moths. 

## Changelog
:cl:
Added USL - United Space Lepidoptera and now spawn as moths.
Removed UPP - United Peoples Party
/:cl:

### CODE REVIEW

File: code/datums/jobs/access.dm [hunk 1]
```diff
@@ -294,33 +294,33 @@
...
 		if("UPP9")
-			. = size ? "ULCDR " : "UPP Lieutenant Commander"
+			. = size ? "ULCDR " : "USL Lieutenant Commander"
 		if("UPP10")
-			. = size ? "UCDR " : "UPP Commander"
+			. = size ? "UCDR " : "USL Commander"
 		if("UPP11")
-			. = size ? "UADM " : "UPP Admiral"
+			. = size ? "UADM " : "USL Admiral"
 		if("UPPC1")
```
Comment (Rohesie) [inline code/datums/jobs/access.dm:318]: You changed the commando standard to `USLC1`
You'll need to change this as well if you want it to work.

File: code/datums/emergency_calls/upp.dm [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
+	to_chat(H, "<B>You grew up on a asteroid with few belongings or luxuries.</B>")
+	to_chat(H, "<B>The family you grew up with were [pick(50;"getting by", 25;"impoverished", 25;"starving")] and you were one of [pick(10;"two", 20;"one", 20;"three", 30;"four", 20;"five")] children.</B>")
+	to_chat(H, "<B>You come from a long line of pirates and quickly joined the local band to improve your living conditions.</B>")
 	to_chat(H, "")
-	to_chat(H, "<B>Today, a TGMC vessel, [SSmapping.configs[SHIP_MAP].map_name], has sent out a distress signal on the orbit of [SSmapping.configs[GROUND_MAP].map_name]. The UPP assault team is authorized and your operation is a go!</b>")
-	to_chat(H, "<B>Eliminate the TGMC force. Do not harm the civilians unless they attack you first.</B>")
+	to_chat(H, "<B>Today, a TGMC vessel, [SSmapping.configs[SHIP_MAP].map_name], has sent out a distress signal on the orbit of [SSmapping.configs[GROUND_MAP].map_name]. Your USL Assault Pirate warband heads out and your stealing begins!</b>")
+	to_chat(H, "<B>Eliminate the TGMC force if necessiary. Do not harm the civilians unless they attack you first.</B>")
```
Comment (Master8688) [inline code/datums/emergency_calls/upp.dm:14]:
```suggestion
	to_chat(H, "<B>Eliminate the TGMC force if necessary. Do not harm the civilians unless they attack you first.</B>")
```

File: code/datums/emergency_calls/upp.dm [hunk 3]
```diff
@@ -1,16 +1,17 @@
...
 	shuttle_id = "distress_upp"
+	spawn_type = /mob/living/carbon/human/species/moth
 
 
 /datum/emergency_call/upp/print_backstory(mob/living/carbon/human/H)
-	to_chat(H, "<B>You grew up in relativly simple family in [pick(75;"a space station", 25;"a famished UPP colony")] with few belongings or luxuries.</B>")
-	to_chat(H, "<B>The family you grew up with were [pick(50;"getting by", 25;"impoverished", 25;"starving")] and you were one of [pick(10;"two", 20;"three", 20;"four", 30;"five", 20;"six")] children.</B>")
-	to_chat(H, "<B>You come from a long line of [pick(40;"crop-harvesters", 20;"soldiers", 20;"factory workers", 5;"scientists", 15;"engineers")], and quickly enlisted to improve your living conditions.</B>")
+	to_chat(H, "<B>You grew up on a asteroid with few belongings or luxuries.</B>")
+	to_chat(H, "<B>The family you grew up with were [pick(50;"getting by", 25;"impoverished", 25;"starving")] and you were one of [pick(10;"two", 20;"one", 20;"three", 30;"four", 20;"five")] children.</B>")
```
Comment (TiviPlus) [inline code/datums/emergency_calls/upp.dm:10]:
```suggestion
	to_chat(H, "<B>The family you grew up with were [pick(50;"getting by", 25;"impoverished", 25;"starving")] and you were one of [pick(10;"two", 20;"three", 30;"four", 20;"five", 20;"six")] children.</B>")
```
one of one children bad

File: code/datums/emergency_calls/upp_commando.dm [hunk 4]
```diff
@@ -1,15 +1,17 @@
...
 	shuttle_id = "distress_upp"
+	spawn_type = /mob/living/carbon/human/species/moth
 
 
 /datum/emergency_call/upp_commando/print_backstory(mob/living/carbon/human/H)
-	to_chat(H, "<B>You grew up in relativly simple family in [pick(75;"Mars", 25;"a famished UPP colony")] with few belongings or luxuries.</B>")
-	to_chat(H, "<B>The family you grew up with were [pick(50;"getting by", 25;"impoverished", 25;"starving")] and you were one of [pick(10;"two", 20;"three", 20;"four", 30;"five", 20;"six")] children.</B>")
-	to_chat(H, "<B>You come from a long line of [pick(40;"crop-harvesters", 20;"soldiers", 20;"factory workers", 5;"scientists", 15;"engineers")], and quickly enlisted to improve your living conditions.</B>")
+	to_chat(H, "<B>You grew up on a asteroid with few belongings or luxuries.</B>")
+	to_chat(H, "<B>The family you grew up with were [pick(50;"getting by", 25;"impoverished", 25;"starving")] and you were one of [pick(10;"two", 20;"one", 20;"three", 30;"four", 20;"five")] children.</B>")
```
Comment (TiviPlus) [inline code/datums/emergency_calls/upp_commando.dm:10]:
```suggestion
	to_chat(H, "<B>The family you grew up with were [pick(50;"getting by", 25;"impoverished", 25;"starving")] and you were one of [pick(10;"two", 20;"three", 30;"four", 20;"five", 20;"six")] children.</B>")
```
ditto

File: code/datums/emergency_calls/upp_commando.dm [hunk 5]
```diff
@@ -1,15 +1,17 @@
...
+	to_chat(H, "<B>You come from a long line of pirates and quickly joined the local band to improve your living conditions.</B>")
 	to_chat(H, "")
-	to_chat(H, "<B>Today, you and your teammates are sent by the United of Progressive Peoples to [SSmapping.configs[SHIP_MAP].map_name] after a long period of cryostasis. Commence the infiltration mission!</b>")
-	to_chat(H, "<B>Eliminate the TGMC force. Do not harm the civilians unless they attack you first.</B>")
+	to_chat(H, "<B>Today, you and your teammates are sent by the United Space Lepidoptera to [SSmapping.configs[SHIP_MAP].map_name] after a long period of lying still in a local asteroid belt. Commence the infiltration mission!</b>")
+	to_chat(H, "<B>Eliminate the TGMC force if necessiary. Do not harm the civilians unless they attack you first.</B>")
```
Comment (TiviPlus) [inline code/datums/emergency_calls/upp_commando.dm:14]:
```suggestion
	to_chat(H, "<B>Eliminate the TGMC force if necessary. Do not harm the civilians unless they attack you first.</B>")
```

**Comment by Rohesie** [comment]
I'm not sure if a direct reference to the Lepidoptera order is the best idea, instead of turning them into some actual alien species, but that's a lore concern.

**Comment by psykzz** [comment]
Does any other lore entries for the UPP need to also be updated?

**Comment by Sibg** [comment]
> Does any other lore entries for the UPP need to also be updated?

The lore will be changed, if it gets merged that is

File: code/datums/emergency_calls/upp.dm [hunk 6]
```diff
@@ -1,16 +1,17 @@
 /datum/emergency_call/upp
-	name = "UPP"
-	probability = 10
+	name = "USL"
+	probability = 0
```
Comment (Rohesie) [inline code/datums/emergency_calls/upp.dm:3]: This means the emergency call won't happen anymore by default. Only through admin intervention.

**Comment by BobSlim** [comment]
*cries in lore*

If these folks are attached to/contracted with an organisation (United Space Lepidoptera) they would more likely be called "Privateers" than pirates.

I would also be a lot more into this if there were deeper, more comprehensive changes from UPP - even theoretical ones. There's nothing really which seems 'pirate' in the UPP kit, I don't think. A one-shot cannon (RPG), a cutlass (sabre), a bottle of rum (rum), the new medHUD eyepatch - that kind of stuff would add greatly to the experience of <s>pirates</s> privateers.

'Sides that, I don't see the point of making them moths. Moths are great, but their general gameplay doesn't deviate from human much. I don't think making them moths gives much to gameplay or roleplay (since most people will play them as if they're basically human).

<s>Pirates</s> Privateers though? Great idea. I see a lot of potential in that.

File: code/datums/emergency_calls/upp.dm [hunk 7]
```diff
@@ -1,16 +1,17 @@
...
 	shuttle_id = "distress_upp"
+	spawn_type = /mob/living/carbon/human/species/moth
 
 
 /datum/emergency_call/upp/print_backstory(mob/living/carbon/human/H)
-	to_chat(H, "<B>You grew up in relativly simple family in [pick(75;"a space station", 25;"a famished UPP colony")] with few belongings or luxuries.</B>")
-	to_chat(H, "<B>The family you grew up with were [pick(50;"getting by", 25;"impoverished", 25;"starving")] and you were one of [pick(10;"two", 20;"three", 20;"four", 30;"five", 20;"six")] children.</B>")
-	to_chat(H, "<B>You come from a long line of [pick(40;"crop-harvesters", 20;"soldiers", 20;"factory workers", 5;"scientists", 15;"engineers")], and quickly enlisted to improve your living conditions.</B>")
+	to_chat(H, "<B>You grew up on a asteroid with few belongings or luxuries.</B>")
```
Comment (spookydonut) [inline code/datums/emergency_calls/upp.dm:9]:
```suggestion
	to_chat(H, "<B>You grew up on an asteroid with few belongings or luxuries.</B>")
```

File: code/datums/emergency_calls/upp_commando.dm [hunk 8]
```diff
@@ -1,15 +1,17 @@
...
+	probability = 0
 	shuttle_id = "distress_upp"
+	spawn_type = /mob/living/carbon/human/species/moth
 
 
 /datum/emergency_call/upp_commando/print_backstory(mob/living/carbon/human/H)
-	to_chat(H, "<B>You grew up in relativly simple family in [pick(75;"Mars", 25;"a famished UPP colony")] with few belongings or luxuries.</B>")
-	to_chat(H, "<B>The family you grew up with were [pick(50;"getting by", 25;"impoverished", 25;"starving")] and you were one of [pick(10;"two", 20;"three", 20;"four", 30;"five", 20;"six")] children.</B>")
-	to_chat(H, "<B>You come from a long line of [pick(40;"crop-harvesters", 20;"soldiers", 20;"factory workers", 5;"scientists", 15;"engineers")], and quickly enlisted to improve your living conditions.</B>")
+	to_chat(H, "<B>You grew up on a asteroid with few belongings or luxuries.</B>")
```
Comment (spookydonut) [inline code/datums/emergency_calls/upp_commando.dm:9]:
```suggestion
	to_chat(H, "<B>You grew up on an asteroid with few belongings or luxuries.</B>")
```

Comment (spookydonut) [reply code/datums/emergency_calls/upp_commando.dm:9]: this can probably just be read from a strings file since its repeated.

**Pull request merged**